sqlglot/dialects/mysql.py

```diff
@@ -163,6 +163,7 @@ class MySQL(Dialect):
     SUPPORTS_USER_DEFINED_TYPES = False
     SUPPORTS_SEMI_ANTI_JOIN = False
     SAFE_DIVISION = True
+    INFINITY = None
```

Comment:
I will try moving this to the `_Dialect.__new__` method in a followup commit to reduce duplication.